build-and-test.ps1

```diff
@@ -0,0 +1,81 @@
+[cmdletbinding()]
+param(
+    [string]$VersionFilter,
+    [string]$OSFilter,
+    [switch]$IsDryRun
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+$buildFilter = "*"
+if (-not [string]::IsNullOrEmpty($VersionFilter))
+{
+    $buildFilter = "$VersionFilter-$buildFilter"
+}
+if (-not [string]::IsNullOrEmpty($OSFilter))
+{
+    if ([string]::IsNullOrEmpty($VersionFilter))
+    {
+        $buildFilter = "$buildFilter-$OSFilter/*"
+    }
+    else
+    {
+        $buildFilter = "$buildFilter$OSFilter/*"
+    }
+}
+
+function Get-ImageBuilder()
+{
+    $imageBuilderImageName = 'microsoft/dotnet-buildtools-prereqs:image-builder-nanoserver-20180103080524'
+    $imageBuilderContainerName = "ImageBuilder-$(Get-Date -Format yyyyMMddhhmmss)"
+
+    $attempt = 0
+    $maxRetries = 5
+    $waitFactor = 6
+    while ($attempt -lt $maxRetries)
+    {
+        try {
+            New-Item -Path "$imageBuilderFolder" -ItemType Directory -Force | Out-Null
+
+            Invoke-Expression "docker pull $imageBuilderImageName"
```

Comment:
As discussed offline, it would be simpler if ImageBuilder binaries were available as a compressed file hosted in Azure storage.  Acquiring ImageBuilder would then be simpler - download and extract,

Reply:
Logged https://github.com/dotnet/docker-tools/issues/67